alacritty_terminal/src/selection.rs

```diff
@@ -41,6 +41,7 @@ pub struct SelectionRange {
 
 impl SelectionRange {
     pub fn new(start: Point, end: Point, is_block: bool) -> Self {
+        assert!(start <= end);
```

Comment:
Did you run into any issues with this, or why add this assert?

Reply:
Since I'm relying on this behavior when damaging. And from what I can see we also rely that start <= end.

Reply:
Hm okay, to what degree do you rely on it? Seems like a `debug_assert!` would be more appropriate? Or does it cause a crash in your damaging logic?

Reply:
I'm iterating from start line to end line. I can work around that by manually sorting them, but I'm not sure that I want to do that, since we do order the range that way when creating it.